config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml

```diff
@@ -346,6 +346,57 @@ periodics:
     testgrid-tab-name: aks-engine-azure-windows-master-staging
     testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
     description: Staging job for new Windows tests on a Kubernetes cluster provided by aks-engine (https://github.com/Azure/aks-engine) on Azure cloud"
+- interval: 3h
+  name: ci-kubernetes-e2e-aks-engine-azure-master-staging-windows-serial
```

Comment:
I thought the plan was to add this for 1.17 first since that release is likely more stable?

Reply:
Agreed. PTAL at the PR.